### ISSUE TITLE
[feature request] connected devices

### ISSUE THREAD

**Issue by josele13** [ubports/ubuntu-touch#1227]
**_Description of the feature:_**

Many times  we give internet to other devices with the access point , to desktop computer, to a laptop, to a tablet or to other mobiles.

I give internet connection by wifi to my computers and relatives outside my home.

When you give wifi you know who you are giving internet connection, but if someone skips your security key can connect to your phone, and have a security problem in your device, this can happen on the street, on the train, in a bar or your neighbor ......

I request that we know how many devices are connected to our Access Point, to verify our security on the phone and in our wallet.

Greetings...

_**Illustrations :**_ 

![screenshot20190910_213048444](https://user-images.githubusercontent.com/52839437/64738961-1077bc00-d4f1-11e9-817c-5d1e87fa0584.jpg)

![screenshot20190910_213048442](https://user-images.githubusercontent.com/52839437/64738965-179eca00-d4f1-11e9-9e03-960f184e4053.jpg)

![screenshot20190910_213048443](https://user-images.githubusercontent.com/52839437/64738980-1ec5d800-d4f1-11e9-9572-5d1844094b9e.jpg)

**Comment by PhoenixLandPirate** [comment]
I think it should be right under the "hotspot"
Perhaps it could look like the vpn options, and just show you what devices are connected, It might look cluttered if you have a lot of devices connected, but I think the majority of people would probably have a maximum of 3, and isn't a very common use case for most people, so I dont think that matters to much.

You could probably disconnect a device the same way you dismiss notifications in the indication menu.

Or as another option, you could have something similar to your screenshot, which just says "x devices connected" that disappears if none are connected, and if you tap, it takes you to system settings where you can get more information or disconnect devices connected.

**Comment by josele13** [comment]
Yeah, I like the way you look at it, I put two photos to show it.

![screenshot20191002_230457995](https://user-images.githubusercontent.com/52839437/66083336-58c45000-e56c-11e9-852f-cf969b6d2b6f.jpg)

![screenshot20191002_230457996](https://user-images.githubusercontent.com/52839437/66083352-62e64e80-e56c-11e9-87e5-13ac4b3ac629.jpg)
